tests/TestHelpers/DB.hs

```diff
@@ -36,8 +36,7 @@ runDB query = do
 createUser :: Text -> Example (Entity User)
 createUser ident = do
     insertEntity User
-        { userFirstName = "John" <> ident
-        , userLastName  = "Smith"
+        { userName      = "John " <> ident <> " Smith"
```

Comment:
Does it break anything to do `"John Smith (" <> ident <> ")"`? I think that'd be better overall, but I understand if you want to keep the changes here to a minimum.